label_studio/data_import/models.py

```diff
@@ -67,6 +67,8 @@ def content(self):
             body = getattr(self, '_file_body')
         else:
             body = self.file.read().decode('utf-8')
+            # TODO: ? filter out malicious/harmful content from file.
```

Comment:
Could we call _allowlist_svg here?

Reply:
yes, but then we are modifying the original file, which can lead to unexpected results. what if our filter takes out too much, users (and us) may be confused.